src/app/auth/login.ts

```diff
@@ -241,7 +241,7 @@ export const loginWithDevice = async ({
     username,
     password,
   })
-  if (res instanceof Error) throw res
+  if (res instanceof Error) return res
```

Comment:
@ntheile we should not throw in the app layer (afair at least, @dolcalmi or @bodymindarts  can confirm - maybe this is my mistake did look at blame)